utils/src/change_format_version.c

```diff
@@ -51,6 +51,12 @@ static int do_change_fmt_vers(struct change_fmt_vers_args *args)
 	int data_fd = -1;
 	int ret;
 
+	if (strcmp(args->meta_device, args->data_device) == 0) {
```

Comment:
Just drop this.  If we had a real problem with having duplicate devices then we'd want something a lot more than string comparison.  It gets very difficult because they could have messed up all sorts of device configuration between path strings and block devices.  But this isn't an important problem -- if you provide the same device then it will always fail the meta/data flag checks.  That's enough to trigger people to stare at their args (/environment) to try and figure out why the devices aren't lining up.

Reply:
Yeah, agreed this isn't needed.